modules/static/grpc-ecosystem/grpc-gateway/initref

```diff
@@ -1 +1 @@
-813bbcd1a729fb6548d3a02ce50c86f2278cb5c0
+v2.11.3
```

Comment:
This is the the closest tag:

```
➜  grpc-gateway git:(813bbcd1) git describe --all 813bbcd1a729fb6548d3a02ce50c86f2278cb5c0
tags/v2.11.3-60-g813bbcd1
```

Here are all the releases: https://github.com/grpc-ecosystem/grpc-gateway/releases